src/components/results/loadflow/load-flow-result-tab.tsx

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React, { FunctionComponent, useState } from 'react';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import { FormattedMessage } from 'react-intl/lib';
+import makeStyles from '@mui/styles/makeStyles';
+import { green, red } from '@mui/material/colors';
+import { LoadflowResultProps } from './load-flow-result.type';
+import { Theme } from '@mui/material';
+import { LoadFlowResult } from './load-flow-result';
+import { useNodeData } from '../../study-container';
+import { fetchLoadFlowResult } from '../../../services/study/loadflow';
+import WaitingLoader from '../../utils/waiting-loader';
+
+export const LoadFlowResultTab: FunctionComponent<LoadflowResultProps> = ({
+    studyUuid,
+    nodeUuid,
+}) => {
+    const useStyles = makeStyles<Theme>((theme) => ({
```

Comment:
```suggestion
    const useStyles = makeStyles(() => ({
```

Reply:
they show an error because it should show an error
those classes don't exist, remove those calls